### PR TITLE
Fix working with GZIP files

### PR DESCRIPTION
**Pull request by Kaspik**
- fix warnings by xcode as `gzopen(path, "rb")` doesn't return object type itself

### CODE REVIEW

**Comment by Kaspik** [comment]
It would be great if anyone else could actually test this, working on Big Sur with XCode 12 doesn't give me much confidence. 😄

File: Source/Controllers/MainViewControllers/SPDatabaseDocument.m [hunk 1]
```diff
@@ -1420,7 +1420,7 @@ - (void) endTask
 		if (queryExecutionTimer) {
 			queryStartDate = [[NSDate alloc] init];
 			[self showQueryExecutionTime];
-			[queryExecutionTimer invalidate], SPClear(queryExecutionTimer);
```
Comment (clayreimann) [inline Source/Controllers/MainViewControllers/SPDatabaseDocument.m:1423]: Did you mean to drop the `SPClear` here?

**Review by clayreimann — COMMENTED** [review]
This seems fine to me. Can you be more specific re: what you think should be tested, and why you're not confident in what you tried?

File: Source/Controllers/MainViewControllers/SPDatabaseDocument.m (hunk 1 above)
Comment (Kaspik) [reply Source/Controllers/MainViewControllers/SPDatabaseDocument.m:1423]: Yes, as you can see one row below, it's there twice. :)

**Comment by Kaspik** [comment]
Re: testing - Just working with GZIP compressed files overall, but I think everything should be okay. :)

**Review by Jason-Morcos — APPROVED** [review]

**Pull request merged**